examples/lines_bars_and_markers/scatter_with_legend.py

```diff
@@ -10,14 +10,15 @@
 from numpy.random import rand
 
 
+fig, ax = plt.subplots()
 for color in ['red', 'green', 'blue']:
     n = 750
     x, y = rand(2, n)
     scale = 200.0 * rand(n)
-    plt.scatter(x, y, c=color, s=scale, label=color,
+    ax.scatter(x, y, c=color, s=scale, label=color,
                 alpha=0.3, edgecolors='none')
```

Comment:
@dopplershift Ahh yes. Good catch. I'll wait for the CI to wrap up before pushing again.